src/index.js

```diff
@@ -67,7 +67,7 @@ function App() {
 
   return (
     <Security {...config} onAuthRequired={authHandler}>
-      <NavBar />
+      {/* <NavBar /> */}
```

Comment:
Commented out code!

Reply:
The Nav was getting in the way so we decided to comment it out. Now, we should have the styling for the global NavBar done.